docs/source/tutorials/videos/index.md

```diff
@@ -0,0 +1,21 @@
+# Nipoppy Tutorial Videos
```

Comment:
The "Nipoppy Tutorial" part feels redundant with the section name and overall project. It could be named Video series